civihr_default_theme/templates/node/webform-calendar.tpl.php

```diff
@@ -3,9 +3,19 @@
 ?>
 <input
   type="text"
-  data-date-format="dd/mm/yyyy"
+  required="<?php $component['required']; ?>"
```

Comment:
These variables are not referenced in the template file so the IDE doesn't know about them. Can you add a doc block at the top of the file mentioning them like
```
/**
 * @var array $component
 *  A description of the component
```